queries/python/highlights.scm

```diff
@@ -29,7 +29,7 @@
 
 ((attribute
     attribute: (identifier) @field)
- (#lua-match? @field "^%u@!.*$"))
+ (#lua-match? @field "^%l.*$"))
```

Comment:
This pattern does not match self._a_b, while the pattern before https://github.com/nvim-treesitter/nvim-treesitter/pull/4686 does match it

Reply:
Good point, one sec

Reply:
Also, regression tests :). I know it's a chore but probably for the better in this case

Reply:
It's not something to really be touched again (only reason was for perf improvements), but if you'd like to add on tests feel free to